x-pack/plugins/observability_solution/observability/server/services/slo/get_burn_rates.ts

```diff
@@ -24,23 +27,57 @@ interface LookbackWindow {
   duration: Duration;
 }
 
-export async function getBurnRates(
-  sloId: string,
-  instanceId: string,
-  windows: LookbackWindow[],
-  services: Services
-) {
+export async function getBurnRates({
+  sloId,
+  windows,
+  instanceId,
+  remoteName,
+  services,
+}: {
+  sloId: string;
+  instanceId: string;
+  remoteName?: string;
+  windows: LookbackWindow[];
+  services: Services;
+}) {
   const { soClient, esClient, logger } = services;
 
   const repository = new KibanaSavedObjectsSLORepository(soClient, logger);
   const sliClient = new DefaultSLIClient(esClient);
-  const slo = await repository.findById(sloId);
 
-  const sliData = await sliClient.fetchSLIDataFrom(slo, instanceId, windows);
+  // find a way to set this based on the sloId used
```

Comment:
Does this still need to be done?

Reply:
No... the discriminator should be if `remoteName` is passed to the function. If not, we should use the local repository to get the SLO definition.